.github/workflows/pull-request.yml

```diff
@@ -51,6 +51,12 @@ jobs:
           kind get kubeconfig > /tmp/kubeconfig
           GOPATH=${GITHUB_WORKSPACE}/go make e2e-sanity
 
+      - name: show abnormal events and all objects in konveyor-forklift
+        if: always()
```

Comment:
so why do we need this statement? wouldn't it make more sense to run this only in some 'debug' mode?

Reply:
well it will execute the step even if the previous steps fails.

Reply:
ack